src/test/java/org/mockitousage/serialization/DeepStubsSerializableTest.java

```diff
@@ -33,6 +35,8 @@ public void should_serialize_and_deserialize_mock_created_with_deep_stubs() thro
 
     @Test
     public void should_serialize_and_deserialize_parameterized_class_mocked_with_deep_stubs() throws Exception {
+        assumeTrue(ClassFileVersion.ofThisVm().isLessThan(ClassFileVersion.JAVA_V9));
```

Comment:
When assumption fails it emits an exception that is ignored by the runner (effectively, later part of this test method is not executed)?